x/bank/client/rest/grpc_query_test.go

```diff
@@ -93,3 +95,76 @@ func (s *IntegrationTestSuite) TestTotalSupplyGRPCHandler() {
 		})
 	}
 }
+
+func (s *IntegrationTestSuite) TestBalancesGRPCHandler() {
+	val := s.network.Validators[0]
+	baseURL := val.APIAddress
+
+	// TODO: need to pass bech32 string instead of base64 encoding string
+	accAddrBase64 := base64.URLEncoding.EncodeToString(val.Address)
+
+	testCases := []struct {
+		name     string
+		url      string
+		headers  map[string]string
+		respType proto.Message
+		expected proto.Message
+	}{
+		{
+			"gRPC total account balance",
+			fmt.Sprintf("%s/cosmos/bank/v1beta1/balances/%s", baseURL, accAddrBase64),
+			map[string]string{
+				grpctypes.GRPCBlockHeightHeader: "1",
+			},
```

Comment:
I think it's clearer to omit these optional headers in general. I would put them only when we're actually testing headers. For the tests here, these headers are imo distracting.

By default it'll just fetch the latest height.